### PR TITLE
Closes #586: Ability to unstack notes from stack

### LINKED ISSUE #586: As a participant, I want to be able to unstack notes from stacks
AC:
- [ ] Participants can unstack stacked notes 
- [ ] Unstacked notes will be displayed in the board again
- [ ] Tests have been added/updated

Design: See v1

Screenshot 2021-09-03 at 10 07 41

### PR DESCRIPTION
**Pull request by Dominik-Weinzierl**
<h2> Description </h2>

Add ability to unstack notes from stacks.

## Checklist 
- [x] Add new button with unstack icon
- [x] Implement unstack behavior
- [x] Add/Update tests

---
<details>
<summary>Visual Changes</summary>

Note dialog

</details>

### CODE REVIEW

File: src/components/NoteDialog/NoteDialog.test.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        votes={[
+          {
+            id: "test-id",
+            board: "test-board",
+            note: "0",
+            user: "test-user",
+          },
+        ]}
+        childrenNotes={[
+          {id: "1", columnId: "TestID", text: "", author: "", parentId: "0", dirty: true, authorName: "", votes: []},
```
Comment (brandstetterm) [inline src/components/NoteDialog/NoteDialog.test.tsx:56]: Above (line 13) you set the column id to "test_column" but here you use "TestID"

File: src/components/NoteDialog/NoteDialog.tsx [hunk 2]
```diff
@@ -48,6 +49,12 @@ const NoteDialog = ({noteId, show, text, authorId, isAdmin, authorName, showAuth
     }
   };
 
+  const onUnstack = (id: string, authorId: string) => {
+    if (editable(authorId)) {
```
Comment (brandstetterm) [inline src/components/NoteDialog/NoteDialog.tsx:53]: I think every participant should be able to unstack notes from stacks

File: src/components/Note/Note.test.tsx [hunk 3]
```diff
@@ -48,11 +48,15 @@ const createNote = (text: string, authorId: string, showAuthors: boolean) => {
...
             id: "test-id",
             board: "test-board",
-            note: "test-note",
+            note: "0",
             user: "test-user",
           },
         ]}
-        childrenNotes={[]}
+        childrenNotes={[
+          {id: "1", columnId: "TestID", text: "", author: "", parentId: "0", dirty: true, authorName: "", votes: []},
```
[the remark this answers is not in the document]
Comment (Dominik-Weinzierl) [reply src/components/Note/Note.test.tsx]: Correct. Should be fixed now.

File: src/components/NoteDialog/NoteDialog.test.tsx (hunk 1 above)
Comment (Dominik-Weinzierl) [reply src/components/NoteDialog/NoteDialog.test.tsx]: Correct. Should be fixed now.

File: src/components/NoteDialog/NoteDialog.tsx (hunk 2 above)
Comment (Dominik-Weinzierl) [reply src/components/NoteDialog/NoteDialog.tsx:53]: Would look like this (normal user who did not create the note). I think this is a good idea.
<details open>
<summary>Unstack notes</summary>

unstack

</details>

Comment (brandstetterm) [reply src/components/NoteDialog/NoteDialog.tsx:53]: 👍🏼

**Review by louiskroener — APPROVED** [review]
LGTM!

**Pull request merged**